libraries/ui-library/src/components/six-select/six-select.tsx

```diff
@@ -312,6 +310,7 @@ export class SixSelect {
         .forEach((option) => (option.checked = hasDeselectedOptions));
       const checkedItems = nonFilteredItems.filter((option) => option.checked).map((option) => option.value);
       this.value = hasDeselectedOptions ? checkedItems : [];
+      this.sixChange.emit({ value: this.value, isSelected: true });
```

Comment:
Shouldnt isSelected be dependent on whether the user has selected an option? Or am I misunderstanding how this works?

Reply:
The `isSelected` property is only `false`, if a user enters a [filter text](https://six-group.github.io/six-webcomponents/components/six-select.html#examples). Otherwise it is always `true`.

So here it is correct to set it to `true`. Should be better documented (or later maybe even renamed).